axelrod/tests/strategies/test_meta.py

```diff
@@ -585,3 +585,77 @@ def test_strategy(self):
         actions = [(C, C), (C, D), (D, C), (D, D), (D, C)]
         self.versus_test(opponent=axelrod.Alternator(),
                          expected_actions=actions)
+
+"""Tests for the Memory Decay strategy"""
+
+class TestMemoryDecay(TestPlayer):
+
+    name = 'MemoryDecay'
+    player = axelrod.meta.MemoryDecay
```

Comment:
This strategy will need to be added to the `axelrod/strategies/__init__.py ` file (where all `meta` strategies get imported), when you've done that you should be able to go with `player = axelrod.MemoryDecay`.